summary_workflows/parameter_codes.md

```diff
@@ -1,23 +1,58 @@
 # Parameter codes
 
-## Input to summary workflows
+Parameter codes uniquely identify what combinations of input files and parameters were used to run an execution workflow. 
```

Comment:
Considering the OpenEBench terminology, it may be better to call these _challenge codes_. What we _currently_ call challenges (identification, quantification, differential usage) would then become _benchmarking events_, and what we currently call _benchmarks_ would become _metrics_ (again?).

It's annoying, but we're unlikely to change the OpenEBench specifications/schemas, which have been around for a while and have some good thinking in them. And it would be good to adopt _one_ consistent terminology.